tests/phpunit/tests/functions/getNonCachedIds.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Test `_get_non_cached_ids()`.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Test class for `_get_non_cached_ids()`.
+ *
+ * @group cache
+ *
+ * @covers ::_get_non_cached_ids
+ */
+class Tests_Functions_GetNonCachedIds extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 57593
+	 *
+	 * @dataProvider data_valid_ids_should_be_returned_as_integers
+	 *
+	 * @param mixed $object_id The object id.
+	 */
+	public function test_valid_ids_should_be_returned_as_integers( $object_id ) {
+		$this->assertSame( array( (int) $object_id ), _get_non_cached_ids( array( $object_id ), 'fake-group' ), 'Object IDs should be returned as integers.' );
+	}
+
+	/**
+	 * Data provider.
+	 *
+	 * @return array
```

Comment:
```suggestion
	 * @return array[]
```

Fix return type error introduced by a Mr CC.